cypress/fixtures/application.json

```diff
@@ -24,5 +24,13 @@
     {
         "repoType": "Subversion",
         "sourceRepo": "https://github.com/konveyor/tackle-testapp"
+    },
+    {
+        "repoType": "Git",
```

Comment:
Please remove this line 28 to 30 . It is already there

Reply:
I can't see another git with http specified in the json file, can you please elaborate ?